app/controllers/shows_controller.rb

```diff
@@ -1,4 +1,10 @@
+# frozen_string_literal: true
+
 class ShowsController < ApplicationController
+  include ShowsHelper
+  
   def index
+    @data_csv = csv_data
```

Comment:
Yes, it makes sense actually. I've renamed it to `parsing_data`. WDYT? 
Just to clarify, the good practice is to name the methods for what they do instead of what they return, right? so in this case is better `parsing_data` than `parsed_data` correct? 👀

Reply:
BTW I'm doing the renaming in PR for issue 3